content/akeneo-event-platform/available-events.md

```diff
@@ -1,4 +1,972 @@
+
+The updates that can trigger this event include the following:
+
+- Update any attribute property and save.
+- Reorder attributes within an attribute group (all attributes in the group are marked as updated).
+
+### Payload
+
+| Name | Type | Description |
+|---|---|---|
+| data.attribute | object | - |
+| data.attribute.code | string | Unique identifier of an Attribute in the PIM. |
+
+> Example of payload
+
+
+```json
+
+{
+  "specversion": "1.0",
+  "id": "b6f4add4-f3d1-4502-8cf0-0f228e054ce5",
+  "type": "com.akeneo.pim.v1.attribute.updated",
+  "source": "pim",
+  "subject": "srnt-whatever",
+  "time": "2024-06-10T17:31:00Z",
+  "datacontenttype": "application/json",
+  "dataschema": "https://event.sdk.akeneo.cloud/com.akeneo.pim.v1.attribute.updated.schema.json",
+  "data": {
+    "attribute": {
+      "code": "an-attribute-code"
+    }
+  }
+}
+```
+
+## com.akeneo.pim.v1.attribute-group.updated
+
+*This event is generated whenever an attribute group is modified in the PIM.*
+
+* Message ID: `com.akeneo.pim.v1.attribute-group.updated`
+* Content type: [application/json](https://www.iana.org/assignments/media-types/application/json)
+
+### Payload
+
+| Name | Type | Description |
+|---|---|---|
+| data.attribute_group | object | - |
+| data.attribute_group.code | string | Unique identifier of an Attribute Group in the PIM. |
+
+> Example of payload
+
+
+```json
+
+{
+  "specversion": "1.0",
+  "id": "b6f4add4-f3d1-4502-8cf0-0f228e054ce5",
+  "type": "com.akeneo.pim.v1.attribute-group.updated",
+  "source": "pim",
+  "subject": "srnt-whatever",
+  "time": "2024-06-10T17:31:00Z",
+  "datacontenttype": "application/json",
+  "dataschema": "https://event.sdk.akeneo.cloud/com.akeneo.pim.v1.attribute-group.updated.schema.json",
+  "data": {
+    "attribute_group": {
+      "code": "an-attribute-group-code"
+    }
+  }
+}
+```
+
+## com.akeneo.pim.v1.attribute-group.deleted
+
+*This event is generated whenever an attribute group is deleted from the PIM.*
+
+* Message ID: `com.akeneo.pim.v1.attribute-group.deleted`
+* Content type: [application/json](https://www.iana.org/assignments/media-types/application/json)
+
+### Payload
+
+| Name | Type | Description |
+|---|---|---|
+| data.attribute_group | object | - |
+| data.attribute_group.code | string | Unique identifier of an Attribute Group in the PIM. |
+
+> Example of payload
+
+
+```json
+
+{
+  "specversion": "1.0",
+  "id": "b6f4add4-f3d1-4502-8cf0-0f228e054ce5",
+  "type": "com.akeneo.pim.v1.attribute-group.deleted",
+  "source": "pim",
+  "subject": "srnt-whatever",
+  "time": "2024-06-10T17:31:00Z",
+  "datacontenttype": "application/json",
+  "dataschema": "https://event.sdk.akeneo.cloud/com.akeneo.pim.v1.attribute-group.deleted.schema.json",
+  "data": {
+    "attribute_group": {
+      "code": "an_attribute_group_code"
+    }
+  }
+}
+```
+
+## com.akeneo.pim.v1.product.created
+
+*This event is generated whenever a new product is created in the PIM.*
```

Comment:
For created event, sometimes it's "created" sometime it's "added" imo we can write create everywhere